languages/rust/exercises/concept/enums/.meta/example.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Clone, PartialEq, Debug)]
+pub enum LogLevel {
+    Info,
+    Warning,
+    Error,
+    Unknown,
+}
+
+pub fn parse_log_level(log: &str) -> LogLevel {
+    if log.starts_with("[ERROR]") {
+        LogLevel::Error
+    } else if log.starts_with("[WARNING]") {
+        LogLevel::Warning
+    } else if log.starts_with("[INFO]") {
+        LogLevel::Info
+    } else {
+        LogLevel::Unknown
+    }
+}
+pub fn output_for_short_log(kind: LogLevel, log: &str) -> String {
+    let code = match kind {
```

Comment:
NOTE: We should be able to declare the integer representation on the `enum` definition?

Reply:
Define the enum with `#[repr(u8)]`; that should let you do `let code = kind as u8`.